test/package/browser/template/package.json

```diff
@@ -8,15 +8,18 @@
     "typecheck": "tsc --project src -noEmit",
     "test-support:server": "ts-node server/server.ts",
     "test": "playwright test",
-    "test:install-deps": "playwright install chromium"
+    "test:install-deps": "playwright install chromium",
+    "test-ct": "playwright test -c playwright-ct.config.ts"
```

Comment:
This `test-ct` naming is a bit mysterious. Also, it would be nice if we could keep it so that the `test` script runs everything. Maybe:

1. rename the existing `test` script to `test:lib`, then
2. rename `test-ct` to `test:hooks`, then
3. reintroduce `test` script which runs `test:lib` and `test:hooks`

Reply:
`test-ct` is a default script name which is added by `npm init playwright@latest -- --ct` command. I think `ct` here stands for "component testing"?

Anyway, your suggested script names are way better, will do that, thanks!